company/how-to-guides-for-staff/how-to-spend-company-money/README.md

```diff
@@ -248,6 +248,8 @@ Team events and meetups should be planned in advance with ample notice and appro
 
 The [Airfare](https://handbook.mattermost.com/education/how-to-guides-for-staff/how-to-spend-company-money#airfare-and-traveling), [Meals and Entertainment](https://handbook.mattermost.com/education/how-to-guides-for-staff/how-to-spend-company-money#meals-and-entertainment), and [Hotel](https://handbook.mattermost.com/education/how-to-guides-for-staff/how-to-spend-company-money#hotel-and-lodging) policies take precedence even in the case of team events or meetups.
 
+Team off sites where travel and overnight stays are required, have a budget of $250 per person per day, which includes lodging, meals and meeting spaces. It does not include travel costs or external partner and customer dinners.
```

Comment:
```suggestion
Team off sites where travel and overnight stays are necessary, we encourage staff to use their best judgment. Historically, the average cost for meals, lodging, and meeting spaces is $250 USD per person per day. Travel costs like airfare and rideshares or external partner and customer dinners are considered an additional expenditure and should seek [manager approval](https://handbook.mattermost.com/operations/finance/staff-member-expenses/how-to-spend-company-money#manager-approval) first. 
```

Reply:
Great update, @TQuock!

Reply:
@amynicol1985 does the above work for you?